test/Benchmarks/src/Main.enso

```diff
@@ -15,14 +17,15 @@ import project.Vector
    sometimes we want to run all benchmarks to ensure our language changes are
    not breaking them.
 main =
-    Collections.bench
-    Equality.bench
-    Json_Bench.bench
-    Natural_Order_Sort.bench
-    Number_Parse.bench
-    Sum.bench
-    Statistics.Main.bench
-    Table.Main.bench
-    Text.Main.bench
-    Time.Main.bench
-    Vector.Main.bench
+    dry_run = Environment.get_or_else "ENSO_BENCHMARK_TEST_DRY_RUN" "False" == "True"
```

Comment:
Thinking about it again, I could probably move this logic to `bench` itself and avoid passing the parameter all over the place.

Reply:
done